src/components/DashboardPage/DashboardPageContainer.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { connect } from "react-redux";
 import DashboardPage from "./DashboardPage";
 import { getAuthUserDashboard } from "../../redux/dashboard-reducer";
+import { State } from "../../redux/redux-store";
 
 class DashboardPageContainer extends React.Component<any> {
```

Comment:
replace any

Reply:
I only can replace any on unknown. Is it right?

Reply:
if no properties required by component then omit generic type altogether `class DashboardPageContainer extends React.Component {`